pip/pip-385-role-anonmyzer.md

```diff
@@ -0,0 +1,120 @@
+# PIP-385: Role Anonymizer for Pulsar Logging
```

Comment:
PIP-385 is already taken. 

Please find another available PIP. The current way to find an available PIP is to search on the dev mailing list discussions whether there's a conflict with the picked PIP number. The first one to open the discussion about the PIP "wins".

You should also make the PR title match the way that we use 
`[feat][pip] PIP-XXX: Role Anonymizer for Pulsar Logging` would be the way to name the PR. Start the thread on the dev mailing list asap after assigning the PR number so that you "win" the possible race condition in assigning the PIP number.